client/src/components/header/header.coffee

```diff
@@ -7,10 +8,15 @@ HeaderBar = React.createFactory require './headerbar.coffee'
 
 Header = React.createClass
   render: ->
+    { login, loginActions, pageLocationActions } = @props
```

Comment:
That was the way they were done in the examples. I got used to it -- kinda liking the bumper space Its been very hard to be consistent though so I think I'll kill the :space_invader: